packages/typespec-go/package.json

```diff
@@ -77,7 +77,7 @@
   "peerDependencies": {
     "@azure-tools/typespec-client-generator-core": "0.56.2",
```

Comment:
This needs to be updated to the same version as above.

Reply:
fixed in [8c448ff](https://github.com/Azure/autorest.go/pull/1585/commits/8c448ff4b20291738f5d792ab9e34e9508ddb915)